resources/lib/WonderfulSubsBrowser.py

```diff
@@ -39,13 +39,13 @@ def _parse_anime_view(self, res):
             res.pop(self._POP_FLAVOR)
 
         if res.get("is_dubbed", None):
-            result.append(utils.allocate_item("%s (Dub)" % base["name"],
+            result.append(utils.allocate_item("%s (Dub)" % base["name"] if not self._POP_FLAVOR else base["name"],
```

Comment:
This code became much more complicated then it should, maybe refactor the whole thing using variable and less code duplication?